main.py

```diff
@@ -25,4 +28,17 @@ async def treeSync(ctx):
     await bot.tree.sync()
     print(bot.tree)
 
+#time the message will send
+#utc is default #10AM EST
+messageTime = datetime.time(hour=14) 
+#method to send all the holidays in discord channel at the specified time
+@tasks.loop(time=messageTime)
+async def NationalDaySend():
+    channel = bot.get_channel(int(CHANNEL)) #this is where the channel id should go!!!
```

Comment:
please add channel id!!! must be type cast as int so you can just keep the `int()` and replace CHANNEL